sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -252,12 +254,23 @@ export class KeysClient {
       delete unflattenedOptions.expires;
       delete unflattenedOptions.requestOptions;
 
+      const span: Span = TracerProxy.getTracer().startSpan(
```

Comment:
If user has not set any tracer, then `TracerProxy.getTracer()` will give the No-op implementation.
The `startSpan()` on the no-op implementation throws an error today meaning the above line would throw an error.

I would imagine the no-op implementation of the tracer to not throw any error

Reply:
Yes. It should not throw an error. I am still working on the NoOp Impl. That is the reason for throwing the error